modules/perception/camera/tools/offline/visualizer.cc

```diff
@@ -1041,28 +1044,29 @@ void Visualizer::Draw2Dand3D_all_info_single_camera(const cv::Mat &img,
     }
 
     cv::line(image_3D, p_proj[0], p_proj[1],
-             colorlistobj[object->track_id % colorlistobj.size()], 2);
+             color, 2);
```

Comment:
Now it fits in one line nicely.

Reply:
Thank you for the review! Changed all lines to one.